apps/communications/contacts/test/marionette/lib/views/contact-details/view.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+/* global module */
+
+var ContactsDetailsAccessor = require('./accessors');
+
+function ContactsDetailsView(client, subject) {
+	this.client = client;
+	this.subject = subject;
+	this.accessors = new ContactsDetailsAccessor(client);
+}
+
+ContactsDetailsView.prototype = {
+	makeFavorited: function() {
+		this.accessors.favoritesButton.tap();
+		return this.waitForFavorited;
```

Comment:
:warning: No need to return the function itself. Calling `this.waitForFavorited()` will do the job.

[More details here](https://github.com/yglazko/gaia/commit/4479b74afae0fcaed81d32d7cf05bc0468566b4f#commitcomment-15614634).